plugins/home/api-report.md

```diff
@@ -0,0 +1,77 @@
+## API Report File for "@backstage/plugin-home"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { LazyExoticComponent } from 'react';
+import { ReactNode } from 'react';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// Warning: (ae-missing-release-tag) "HomepageCompositionRoot" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const HomepageCompositionRoot: (props: {
+  title?: string | undefined;
+  children?: ReactNode;
+}) => JSX.Element;
+
+// Warning: (ae-missing-release-tag) "homePlugin" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const homePlugin: BackstagePlugin<
+  {
+    root: RouteRef<undefined>;
+  },
+  {}
+>;
+
+// Warning: (ae-missing-release-tag) "RandomJokeHomePageComponent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const RandomJokeHomePageComponent: (
+  props: {
+    Renderer?:
+      | ((
+          props: {
+            title: string;
+          } & {
+            Content: LazyExoticComponent<(props: any) => JSX.Element>;
+            Actions?:
+              | LazyExoticComponent<(props: any) => JSX.Element>
+              | undefined;
+            Settings?:
+              | LazyExoticComponent<(props: any) => JSX.Element>
+              | undefined;
+            ContextProvider?:
+              | LazyExoticComponent<(props: any) => JSX.Element>
+              | undefined;
+          },
+        ) => JSX.Element)
+      | undefined;
+  } & {
+    title?: string | undefined;
+  } & {
+    defaultCategory?: 'any' | 'programming' | undefined;
+  },
+) => JSX.Element;
+
+// Warning: (ae-missing-release-tag) "SettingsModal" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const SettingsModal: ({
```

Comment:
Yes, it turned out to be useful when developing other renderers, that probably want to use the same settings modal. Perhaps this can be built in implicitly in the future, without having to manually assign it as being responsible for settings. For now I think it makes sense to export it though.